### PR TITLE
add release-tool workflow to generate release pr

### PR DESCRIPTION
**Pull request by cw-Guo**
### What this PR does / why we need it:
Automate the release process, it will create a release pr (step 2, example #1354 ) automatically. 
We need another automation for step1, example #1343 

### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by cw-Guo** [comment]
This only covers the step 2 (#1354 )

Here is an example pr created by this workflow: https://github.com/cw-Guo/fluent-operator/pull/5/files

**Review by wenchajun — APPROVED** [review]

File: .github/workflows/release-tool.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          git config user.email "actions@github.com"
+
+      - name: Update version in manifests
+        run: |
+          sed -i 's/fluent-operator:latest/fluent-operator:${{ github.event.inputs.version }}/' manifests/setup/fluent-operator-deployment.yaml
+          sed -i 's/fluent-operator:latest/fluent-operator:${{ github.event.inputs.version }}/' manifests/setup/setup.yaml
+
+      - name: Update version in values.yaml
+        run: |
+          sed -i '/repository: "kubesphere\/fluent-operator"/!b;n;s/tag: "latest"/tag: "${{ github.event.inputs.version }}"/' charts/fluent-operator/values.yaml
```
Comment (wenchajun) [inline .github/workflows/release-tool.yaml:66]: We may consider removing the `kubesphere` prefix later.

**Comment by wenchajun** [comment]
Thank you for your contribution.

**Comment by benjaminhuo** [comment]
That's great!

**Pull request merged**